docs/helm-chart/production-guide.rst

```diff
@@ -184,6 +184,44 @@ To use an external StatsD instance:
       statsd_host: ...
       statsd_port: ...
 
+Datadog
+^^^^^^^
+If you are using a Datadog agent in your environment, this will enable Airflow to export metrics to the Datadog agent.
+NOTE: The below is for version 2.x.x of Airflow.  In Airflow 1.x.x the swap all instances of ``metrics`` for ``scheduler``
+
+.. code-block:: yaml
+
+  statsd:
+    enabled: false
+  config:
+    metrics:
+      statsd_on: true
+      statsd_port: ``<port>``
+      statsd_prefix: airflow
```

Comment:
```suggestion
```

We don't need to show the default, do we?

Reply:
I'm not sure on this one.

I did it for sake of showing the full block of configuration, didn't want someone to think it needed to be removed since it's already a part of the helm chart.